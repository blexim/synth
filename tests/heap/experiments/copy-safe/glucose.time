WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 20
0.80user 0.02system 0:00.83elapsed 99%CPU (0avgtext+0avgdata 48196maxresident)k
0inputs+16outputs (0major+15999minor)pagefaults 0swaps
