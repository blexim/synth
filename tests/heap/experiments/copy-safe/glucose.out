c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:         88662                                                                   |
c |  Number of clauses:          280871                                                                   |
c |  Parse time:                   0.10 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           2.57 Mb                                                                |
c |  Simplification time:          0.29 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c |       48         0      208 |    6624    44889   179593 |     2     6100      323     3753 | 27.132 % |
c =========================================================================================================
c restarts              : 48 (252 conflicts in avg)
c blocked restarts      : 0 (multiple: 0) 
c last block at restart : 0
c nb ReduceDB           : 2
c nb removed Clauses    : 3753
c nb learnts DL2        : 375
c nb learnts size 2     : 170
c nb learnts size 1     : 27
c conflicts             : 12126          (15032 /sec)
c decisions             : 23064          (0.00 % random) (28592 /sec)
c propagations          : 2891696        (3584786 /sec)
c conflict literals     : 183471         (33.35 % deleted)
c nb reduced Clauses    : 205
c CPU time              : 0.806658 s

s UNSATISFIABLE
